src/test/scala/edu/ie3/mobsim/model/TripSimulationTestData.scala

```diff
@@ -0,0 +1,264 @@
+    "car_3",
+    givenModel,
+    givenHomePoi,
+    givenWorkPoi,
+    givenSimulationStart,
+    givenFirstDeparture,
+    isChargingAtHomePossible
+  )
+
+  def ev4: ElectricVehicle = ElectricVehicle.buildEv(
+    "car_4",
+    givenModel,
+    givenHomePoi,
+    givenWorkPoi,
+    givenSimulationStart,
+    givenFirstDeparture,
+    isChargingAtHomePossible
+  )
+
+  def ev5: ElectricVehicle = ElectricVehicle.buildEv(
+    "car_5",
+    givenModel,
+    givenHomePoi,
+    givenWorkPoi,
+    givenSimulationStart,
+    givenFirstDeparture,
+    isChargingAtHomePossible
+  )
+
+  protected val chargingStations: Set[ChargingStation] =
+    Set(cs0, cs1, cs2, cs3, cs4, cs5, cs6)
+
+  private val poiData: Seq[PointOfInterest] = Seq(
+    poiHome,
+    workPoi,
+    bbpgPoi,
+    culturePoi,
+    medicinalPoi,
+    other_shopPoi,
+    religiousPoi,
+    restaurantPoi,
+    servicePoi,
+    sportsPoi,
+    supermarketPoi,
+    charging_hub_townPoi,
+    charging_hub_highwayPoi
+  )
+
+  protected val plannedDestinationPoi: PointOfInterest = poiData(11)
+
+  protected val plannedStoredEnergyEndOfTrip: ComparableQuantity[Energy] = half
+  protected val plannedDestinationPoiType: PoiTypeDictionary.Value =
+    PoiTypeDictionary.LEISURE
+  protected val plannedDestinationCategoricalLocation
+      : CategoricalLocationDictionary.Value =
+    CategoricalLocationDictionary.CULTURE
+  protected val plannedParkingTimeStart: ZonedDateTime =
+    givenSimulationStart.plusMinutes(30)
+  protected val plannedDepartureTime: ZonedDateTime =
+    givenSimulationStart.plusHours(2)
+
+  protected val pois: Set[PointOfInterest] = poiData.toSet
+
+  protected val poisWithSizes: Map[
+    PoiEnums.CategoricalLocationDictionary.Value,
+    ProbabilityDensityFunction[PointOfInterest]
+  ] = PoiUtils.createPoiPdf(
+    pois.map { poi =>
+      poi.categoricalLocation -> Set(poi)
+    }.toMap
+  )
+
+  private val speedFunction: SpeedFunction =
+    SpeedFunction(50, 0, Quantities.getQuantity(50, KILOMETRE_PER_HOUR))
+
+  private val speedMap: Map[Int, SpeedFunction] =
+    Range(0, 12).map(_ -> speedFunction).toMap
+
+  protected val speed: DrivingSpeed = DrivingSpeed(speedMap, speedMap, speedMap)
+
+  protected val storedEnergyValue: ComparableQuantity[Energy] =
+    Quantities.getQuantity(20, PowerSystemUnits.KILOWATTHOUR)
+
+  protected val maxDistance: ComparableQuantity[Length] =
+    Quantities.getQuantity(5000, METRE)
+
+  protected val ioUtils: IoUtils = IoUtils(
+    new java.io.File(
+      "."
+    ).getCanonicalPath + File.separator + "out" + File.separator,
+    "movements",
+    "evs",
+    "evcs",
+    "positions",
+    "pois"
+  )
+
+  private val basePath: String = Seq(
+    Paths.get("").toAbsolutePath.toString,
+    "src",
+    "test",
+    "resources",
+    "edu",
+    "ie3",
+    "mobsim",
+    "model"
+  ).mkString(File.separator) + File.separator
+  private val catLocFile: String = basePath + "categorical_location.csv"
```

Comment:
You can use `getClass.getResource()` here, that's the purpose of putting these files in the package directory. I just saw that you had implemented it in this way before, why did you revert it?

Reply:
Ah, I see. One thing you could do is provide a `object ModelResourceLoader` inside `edu.ie3.mobsim.model`, which provides `catLocFile`, `drivingSpeedFile`, etc. to all tests including to `MobilitySimulatorSpec` in #53